.config.yml

```diff
@@ -113,6 +113,7 @@ custom:
     last_email_fe: "http://localhost:3000/email/last-email"
     last_email_fo: "http://localhost:3002/fo/email/last-email"
     last_email_bo: "http://localhost:8001/bo/email/last-email"
+    mock_enabled: "http://localhost:3002/fo/mocks/company/00445790"
```

Comment:
Although we will aim to set all mocks as on or off together, it is possible for the Companies House and WorldPay mocks to be set independently. Should we rename this to `ch_mock_enabled` so that we're less likely to get confused?